cluster/src/main/java/com/hartwig/pipeline/reruns/StartingPoint.java

```diff
@@ -50,9 +60,17 @@ enum StartingPoints {
                         Cobalt.NAMESPACE,
                         Amber.NAMESPACE,
                         SageConfiguration.SAGE_GERMLINE_NAMESPACE))),
-        GRIPSS_COMPLETE(concat(CALLING_COMPLETE.namespaces,
-                List.of(SOMATIC_NAMESPACE, GERMLINE_NAMESPACE))),
-        PURPLE_COMPLETE(concat(GRIPSS_COMPLETE.namespaces, List.of(Purple.NAMESPACE)));
+        GRIPSS_COMPLETE(concat(CALLING_COMPLETE.namespaces, List.of(SOMATIC_NAMESPACE, GERMLINE_NAMESPACE))),
+        PURPLE_COMPLETE(concat(GRIPSS_COMPLETE.namespaces, List.of(PaveSomatic.NAMESPACE, PaveGermline.NAMESPACE, Purple.NAMESPACE))),
```

Comment:
I think it probably makes more sense to move PAVE up into CALLER_COMPLETE or its own starting point. Previously gene annotations were part of the somatic caller.